pynicotine/__init__.py

```diff
@@ -105,6 +106,7 @@ def check_arguments():
 
     core.cli_interface_address = args.bindip
     core.cli_listen_port = args.port
+    core.isolated_mode = args.isolated_mode
```

Comment:
... `or (os.environ.get("GDK_BACKEND") == "broadway")`

Reply:
I added the --isolated-mode flag because I didn't want to tie the mode to a specific backend. GTK-specific details don't belong in the core.